NuGet.config

```diff
@@ -17,6 +17,7 @@
     <add key="dotnet5-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5-transport/nuget/v3/index.json" />
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
     <add key="dotnet-coreclr" value="https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json" />
+    <add key="eerhardt-test" value="https://www.myget.org/F/eerhardt-testfeed/api/v3/index.json" />
```

Comment:
this should probably be removed.

Reply:
Yep. These are temporarily necessary to get the fix for https://github.com/dotnet/arcade/pull/4852. Without it the code won’t build. I’ll revert that temporary commit before merging.

Reply:
Can you instead just set the nullable context to annotations for the builds in question?

Reply:
The arcade change has now been merged into dotnet/runtime master. I have reverted the temporary commit to use the local arcade.  There is no need for a workaround.

Reply:
> There is no need for a workaround.

Can the other workaround cited be removed as well then? Or was that a different issue?

Reply:
What other workaround cited?

There was an issue with how we generate `.notsupported.cs` files w.r.t. nullable annotations. See https://github.com/dotnet/arcade/pull/4852. I was using a locally built arcade that contained the fix in order to test my changes here. Now that the arcade change has flown into dotnet/runtime, I have removed all workarounds.

Reply:
The example I cited above in https://github.com/dotnet/runtime/pull/32343#discussion_r379707979 from System.Security.Principal.Windows.

Reply:
Ah, I missed that was a different project. I thought it was for this project. Yes, I can remove the System.Security.Principal.Windows workaround.

Reply:
Thanks.

Reply:
It turns out the workaround in System.Security.Principal.Windows is a different issue - https://github.com/dotnet/arcade/issues/4722. 

https://github.com/dotnet/runtime/blob/6ce761095b57ed38242c8140bc4f1d6da7251567/src/libraries/System.Security.Principal.Windows/ref/System.Security.Principal.Windows.cs#L268

I have updated the comment to point to the underlying arcade issue.